[msys2] CMake configuring failure with 3.0.2 version

### Prerequisite Checklist

- [x] I searched for [existing issues](https://github.com/search?q=repo%3ASFML%2FSFML&type=issues) to prevent duplicates
- [x] I searched for [existing discussions on the forum](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fen.sfml-dev.org) to prevent duplicates
- [x] I am here to report an issue and not to just ask a question or look for help (use the [forum](https://en.sfml-dev.org/forums/index.php#c3) or [Discord](https://discord.gg/nr4X7Fh) instead)

### Describe your issue here

when trying to configure cmake for SFML 3.0.2 I get this error
```
CMake Error: File C:/msys64/home/maksa/forks/MINGW-packages/mingw-w64-sfml/src/SFML-3.0.2/tools/pkg-config/sfml-main.pc.in does not exist.
CMake Error at cmake/Macros.cmake:236 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  src/SFML/Main/CMakeLists.txt:16 (sfml_add_library)
```
and sfml-main.pc.in is actually missing

### Your Environment

- OS / distro / window manager: MSYS2
- SFML version: 3.0.2
- Compiler / toolchain: CLANG64 environment
- Special compiler / CMake flags: 
```
-GNinja
-DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX}
-DSFML_BUILD_EXAMPLES=ON
-DSFML_BUILD_DOC=ON
-DSFML_INSTALL_PKGCONFIG_FILES=ON
-DSFML_PKGCONFIG_INSTALL_PREFIX=${MINGW_PREFIX}/lib/pkgconfig
-DBUILD_SHARED_LIBS=ON
-DSFML_USE_SYSTEM_DEPS=ON
-S SFML-${pkgver}
-B build-${MSYSTEM}-shared
```

### Steps to reproduce

described above...

### Expected behavior

successful configured build file

### Actual behavior

^

--- a/cmake/Macros.cmake
+++ b/cmake/Macros.cmake
@@ -238,7 +238,7 @@ macro(sfml_add_library module)
             FRAMEWORK DESTINATION "." COMPONENT bin)
 
     # install pkgconfig
-    if(SFML_INSTALL_PKGCONFIG_FILES)
+    if(SFML_INSTALL_PKGCONFIG_FILES AND NOT ${target} STREQUAL "sfml-main")
         configure_file(
             "${PROJECT_SOURCE_DIR}/tools/pkg-config/${target}.pc.in"
             "${CMAKE_CURRENT_BINARY_DIR}/tools/pkg-config/${target}.pc"

